MonoGame.Framework.Content.Pipeline/Graphics/FontDescriptionStyle.cs

```diff
@@ -14,6 +14,11 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Graphics
     public enum FontDescriptionStyle
     {
         /// <summary>
+        /// Normal text.
+        /// </summary>
+        Regular=0,
```

Comment:
Yesterday I got fed up with "we have to compare this with what XNA does", so this morning I have installed the XNA tooling and run the texts to see what happens.
Regarding this enum values I was right, this are the tests result.
![fontdescriptionstyle](https://cloud.githubusercontent.com/assets/11319966/13329327/8a1a15b2-dbf3-11e5-8a39-7cb6135b7aa3.PNG)